### DISCUSSION TITLE
Transfer stopped mid way?

### DISCUSSION THREAD

**Discussion by not4programmer** [localsend/localsend#1599] [Discussion (English, International)] [answered]
LocalSend v1.1.3.1 for Windows 10
LocalSend v1.15.3 for Android 14, Samsung A53

just happened several days ago, any files, be it single files or multiple, there's always one that stopped in the middle of transfer

no error, no failure, nothing, just stop transferring.

different wifi and laptop, same result (I have no 2nd android device unfortunately)
has tried changing private DNS (as I use adguard dns) to automatic or off, also same result

this just happened in recent transfers, a month or so ago it works fine without changing anything, even with PrivateDNS on

**Comment by Tienisto** [comment, marked as answer]
I have also noticed this. Have you set a custom path? Since 1.15.2, LocalSend uses content URIs to save files (path starts with `content://`). I suspect that this causes some issues

**Reply by not4programmer** [reply]
yes this is likely the culprit. I just clear the app cache to reset the default path and it works again. thanks
